From: Kemeng Shi <shikemeng@huaweicloud.com>
Subject: mm: swap: remove stale comment of swap_reclaim_full_clusters()
Date: Sun, 23 Feb 2025 00:08:50 +0800

swap_reclaim_full_clusters() has no return value now, just remove the
stale comment which says swap_reclaim_full_clusters() wil return a bool
value.

Link: https://lkml.kernel.org/r/20250222160850.505274-7-shikemeng@huaweicloud.com
Signed-off-by: Kemeng Shi <shikemeng@huaweicloud.com>
Cc: Kairui Song <ryncsn@gmail.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 mm/swapfile.c |    1 -
 1 file changed, 1 deletion(-)

--- a/mm/swapfile.c~mm-swap-remove-stale-comment-of-swap_reclaim_full_clusters
+++ a/mm/swapfile.c
@@ -820,7 +820,6 @@ out:
 	return found;
 }
 
-/* Return true if reclaimed a whole cluster */
 static void swap_reclaim_full_clusters(struct swap_info_struct *si, bool force)
 {
 	long to_scan = 1;
_
